IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoad/PropertySets/Pset_PavementCommon/Documentation.md

```diff
@@ -0,0 +1 @@
+Properties for pavement.
```

Comment:
As above possible location for clarifications on nominalThickness and possibly nominalLength.